Makefile

```diff
@@ -50,3 +50,11 @@ clear-cache:
 	docker rm -vf $$(docker ps -a -q) 2>/dev/null ; \
 	docker rmi -f $$(docker images -a -q) 2>/dev/null ; \
 	docker volume rm $$(docker volume ls -q) 2>/dev/null ; true
+
+experiment=*
+
+stop-trials:
+	docker rm -vf $$(docker ps -f "name=fuzzbench-r-$(experiment)-[0-9]+$$" -q) 2>/dev/null ; true
```

Comment:
consider using full flags in code e.g. `rm --force --verbose (??)`, and avoid abbreviations for readability. that way the reader don't need to look up e.g. what `-v` is